src/ProjectTemplates/Quantum.App1/Quantum.App1.csproj.v.template

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.Quantum.Sdk/#NUGET_VERSION#" InitialTargets="RecompileOnChange">
```

Comment:
Will InitialTargets still be needed with the non-draft version of the SDK?

Reply:
No, got rid of it - now we properly add all QsharpCompile items to a suitable item group for msbuild to treat them as source files.